l10n_ch_payment_slip/payment_slip.py

```diff
@@ -483,9 +483,6 @@ def _draw_address(self, canvas, print_settings, initial_position, font,
         text.moveCursor(0.0, font.size)
 
         address_lines = com_partner.contact_address.split("\n")
-        if com_partner.country_id:
```

Comment:
> For your specific case, you should at least test if country is part of self.country_id.address_format

Hi @yvaucher , in the first place, I made the change as you suggest, with something like:

```
if 'country' in com_partner.country_id.address_format:
    del address_lines[-1]
```
then I looked at version 9:
https://github.com/OCA/l10n-switzerland/blob/9.0/l10n_ch_payment_slip/models/payment_slip.py#L471
and made this PR, assuming that, if users don't want to print the country , they just remove country from address_format.

That being said, which solution do you suggest?
Thanks